sdk/formrecognizer/azure-ai-formrecognizer/samples/async_samples/sample_differentiate_custom_forms_with_labeled_and_unlabeled_models_async.py

```diff
@@ -0,0 +1,97 @@
+# coding: utf-8
```

Comment:
Not sure how samples will be listed, but given that this is an advanced sample that comes first alphabetically, would it make sense to add a number (e.g., sample{n}_<name>.py) to order it later in the directory listing?

Reply:
I think since there aren't many samples and I haven't seen this pattern in other python samples, I won't make this change for now. Might revisit if we add more samples. Thanks!